db/migrate/20220324220703_create_users.rb

```diff
@@ -0,0 +1,12 @@
+class CreateUsers < ActiveRecord::Migration[7.0]
+  def change
+    create_table :users do |t|
+      t.string :name
+      t.string :photo
+      t.string :bio
```

Comment:
- [ ] Following the requirement, `:bio` should be of type `text` and `string`. It will be nice to fix this.

Reply:
This issue is well resolved! ✔️